torchtext/experimental/vocab.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+import torch
+import torch.nn as nn
+
+logger = logging.getLogger(__name__)
+
+
+class Vocab(nn.Module):
+    r"""Creates a vocab object which maps tokens to indices.
+
+    Arguments:
+        ordered_dict (collections.OrderedDict): object holding the frequencies of each token found in the data.
```

Comment:
I think we can be more explicit here. For example, what happen if there are tokens with same frequency.

Reply:
Hmm so refer to my comments above for respecting the ordering of the dictionary that is passed in.

Reply:
Does this particular class documentation make the explicit to the user?

Reply:
Sure I think that's a good idea. I just added 2 lines explaining this